.github/workflows/check.yml

```
@@ -11,7 +11,9 @@ permissions: read-all
 jobs:
   audit:
     name: Audit
-    uses: ericcornelissen/git-tag-annotation-action/.github/workflows/reusable-audit.yml@main
+    uses: ericcornelissen/git-tag-annotation-action/.github/workflows/reusable-audit.yml@381-audit-script
+    with:
+      refs: '["v2"]'
```

Revert this prior to merging

```suggestion
    uses: ericcornelissen/git-tag-annotation-action/.github/workflows/reusable-audit.yml@main
```